lib/SimpleRenderer.js

```diff
@@ -0,0 +1,22 @@
+var fs = require('fs');
```

Comment:
I kind of think 

> While OSX and Windows will treat 'myclass.js' and 'MyClass.js' equivalently, Linux won't.

means use upper and lower case, but never rely on case insensitive requires. Eg if we make sure to never do this: `require('simplerenderer')` then we're OK. Even better if we can lint it.